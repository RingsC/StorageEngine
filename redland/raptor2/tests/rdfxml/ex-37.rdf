<?xml version="1.0"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:ex="http://example.org/ns#">

  <!-- Test rdf:type with a (URI) value gets propogated to the
       generated blank node
  -->

  <rdf:Description rdf:about="http://example.org/node">
    <ex:prop rdf:type="http://example.org/ns#Thing"></ex:prop>
  </rdf:Description> 
</rdf:RDF>
