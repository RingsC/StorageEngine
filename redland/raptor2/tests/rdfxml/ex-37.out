_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/ns#Thing> .
<http://example.org/node> <http://example.org/ns#prop> _:genid1 .
